pages/school-registration/index.tsx

```diff
@@ -44,6 +46,9 @@ const DirApps = () => {
             height={85}
             image="awards.jpg"
         />
+        <ComponentWrapper margins={"auto"} pad={"0 16rem 0 16rem"}>
```

Comment:
if a component is wrapped in a single component wrapper then inside the component itself the parent tag should be component wrapper so when its used it can just be itself <RegistrationDetails /> with no wrapper